.travis.yml

```diff
@@ -3,4 +3,4 @@ sudo: false
 dist: trusty
 
 script:
-  - npm run lint .
+  - npm run lint . && npm test
```

Comment:
Er, I thought that's what you wanted. `&&` doesn't execute the right side if the left side exits non-zero.